docs/config/lua/wezterm/canonical_path.md

```diff
@@ -0,0 +1,43 @@
+---
+title: wezterm.canonical_path
+tags:
+ - utility
+ - filesystem
+---
+# `wezterm.canonical_path(path)`
+
+{{since('nightly')}}
+
+This function returns a string with the canonical form of a path if it exists.
+The returned path is in absolute form with all intermediate components normalized
+and symbolic links resolved.
+Due to limitations in the lua bindings, all of the paths
+must be able to be represented as UTF-8 or this function will generate an
+error.
+
```

Comment:
For SEO purposes, I think it would be good to have a sentence here that mentions both `realpath` and `readlink`, which are similar operations that people may be used to in other languages/environments.